app-editors/remarkable/remarkable-1.9.0_pre20210320-r3.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
Remove 3.9, it's no longer supported and causes pkgcheck warnings.

Reply:
I'll remove it, but pkgcheck 0.10.29 does not give a python related warning.

```
martin@bln9716cm ~/swbuild/gentoo-tree/gentoo/app-editors/remarkable $ pkgcheck scan --net
app-editors/remarkable
  UnstableOnly: for arches: [ amd64, arm64, x86 ], all versions are unstable: [ 1.9.0_pre20210320-r3 ]
```

Reply:
You need to run it with `--commits`.